provider/lxd/environ_test.go

```diff
@@ -132,17 +132,13 @@ func (s *environSuite) TestDestroy(c *gc.C) {
 	s.Client.Volumes = map[string][]api.StorageVolume{
 		"juju": {{
 			Name: "not-ours",
-			StorageVolumePut: api.StorageVolumePut{
-				Config: map[string]string{
-					"user.juju-model-uuid": "other",
-				},
+			Config: map[string]string{
+				"user.juju-model-uuid": "other",
```

Comment:
I'm assuming that most of these changes are just because they flattened their type structure. 

Looking at the old LXD api:
```
type CertificatesPost struct {
        CertificatePut `yaml:",inline"`
        
```

It would appear that it was already an embedded struct vs a separate struct, and yaml inline would indicate that the fields are getting serialized to the same structure in both cases. (because it was embedded, we probably could have used the new syntax anyway.)